### DISCUSSION TITLE
Hacked account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#45244] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello I am not able to delete this account. Is it possible I am under another account or organization and am unable to see it? If so, how do I remove it and or disconnect from. It sends me emails about new IP addresses and name changes….

**Comment by queenofcorgis** [comment]
Hi @VOSU2013 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
